### PR TITLE
Update guide to remove vnx related information

### PR DESCRIPTION
**Pull request by wendyzheng0**
New Cinder driver doesn't support vnx any more. Remove information about vnx from the guide.

### CODE REVIEW

File: cinder/README.md [hunk 1]
```diff
@@ -125,23 +125,23 @@ unity_storage_pool_names=
...
-`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` can be used to configure both unity and vnx as multi-backends.
+`cinder-dellemc-unity-config.yaml` and `cinder-xxx-config.yaml` can be used to configure unity and another storage system as multi-backends.
 
 ```bash
 (undercloud) $ openstack overcloud deploy --templates \
   -e /home/stack/templates/containers-prepare-parameter.yaml \
   -e /home/stack/templates/custom-dellemc-container.yaml \
   -e <other templates> \
-  -e /home/stack/templates/cinder-backend-dellemc-vnx.yaml \
+  -e /home/stack/templates/cinder-backend-xxx.yaml \
```
Comment (xiedenglu) [inline cinder/README.md:137]: We might need keep "dellemc" or "dell" in it.

File: cinder/README.md [hunk 2]
```diff
@@ -125,23 +125,23 @@ unity_storage_pool_names=
 
 ## Multiple Back-ends
 
-To configure multiple backends of two different storage types you can combine pass both environment files on the same deploy command as below.
+To configure multiple backends of two different storage types you can combine pass more than one environment files on the same deploy command as below.
 
-`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` can be used to configure both unity and vnx as multi-backends.
+`cinder-dellemc-unity-config.yaml` and `cinder-xxx-config.yaml` can be used to configure unity and another storage system as multi-backends.
```
Comment (xiedenglu) [inline cinder/README.md:130]: We might need keep "dellemc" or "dell" in it.

File: cinder/README.md (hunk 1 above)
Comment (wendyzheng0) [reply cinder/README.md:137]: This is just an example, no need to be dell's product.

File: cinder/README.md (hunk 2 above)
Comment (wendyzheng0) [reply cinder/README.md:130]: Similar comment.

**Pull request merged**